src/algebra/quotient.lean

```diff
@@ -0,0 +1,53 @@
+/-
+Copyright (c) 2021 Anne Baanen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen
+-/
+
+import data.set_like.basic
+
+/-!
+# Algebraic quotients
+
+This file defines notation for algebraic quotients, e.g. quotient groups `G /// H`,
+quotient modules `M /// N` and ideal quotients `R /// I`.
+
+The actual quotient structures are defined in the following files:
+ * quotient group: `src/group_theory/quotient_group.lean`
+ * quotient module: `src/linear_algebra/quotient.lean`
+ * quotient ring: `src/ring_theory/ideal/quotient.lean`
+
+## Notations
+
+The following notation is introduced:
+
+* `G /// H` stands for the quotient of the type `G` by the `set_like` element H.
+  To implement this notation for other quotients, you should provide a `has_quotient` instance.
```

Comment:
Note that since `G` can usually be inferred from `H`, `_ /// H` can also be used, but this is less readable.